women-in-tech-resources/resources-list.md

```diff
@@ -164,7 +164,8 @@ If this is your first time looking at the Women in Tech Resources List, spend 30
 - [Lets Intern](https://www.letsintern.com/)
 - [Intern Supply](https://intern.supply/)
 - [Techgig](https://www.techgig.com/)
-
+- [Twenty19](http://www.twenty19.com/)
+- [Hello Intern](https://www.hellointern.com/)
```

Comment:
Twenty19 looks good, but Hello Intern doesn’t seem to have many resources. Can you take it out?

Reply:
Resolved!